kornia/geometry/boxes.py

```diff
@@ -340,10 +429,18 @@ def transform_boxes_(self, M: torch.Tensor) -> "Boxes":
         """Inplace version of :func:`Boxes.transform_boxes`"""
         return self.transform_boxes(M, inplace=True)
 
+    def _set_source_mode(self, mode: str) -> None:
+        """Set the default mode for functions like `to_tensor`."""
+        self._mode = mode
+
     @property
     def _is_batch(self) -> bool:
```

Comment:
```suggestion
    def is_batch(self) -> bool:
```

Reply:
or `is_batched`

Reply:
Not sure about it. Why would a user want to know if it is batched when we are always returning the input dimensions?